Makefile

```diff
@@ -317,6 +317,33 @@ clean-volumes:
 compose:
 	$(foreach MODULE, $(CANDIG_MODULES), $(MAKE) compose-$(MODULE);)
 
+#-- Temp --
+compose-authz:
+	docker-compose -f $(DIR)/lib/keycloak/docker-compose.yml up -d 2>&1
+	docker-compose -f $(DIR)/lib/vault/docker-compose.yml up -d 2>&1
+	docker-compose -f $(DIR)/lib/tyk/docker-compose.yml up -d 2>&1
+compose-authz-down:
+	docker-compose -f $(DIR)/lib/keycloak/docker-compose.yml down
+	docker-compose -f $(DIR)/lib/vault/docker-compose.yml down
+	docker-compose -f $(DIR)/lib/tyk/docker-compose.yml down
+compose-authz-clean: compose-authz-down \
+	# needs sudo to run;
+	./etc/setup/scripts/sudo_check.sh
+
+	# clean keycloak
+	sudo rm -r $(DIR)/lib/keycloak/volumes/* & 2>&1
```

Comment:
use $(DIR)/tmp/data/auth instead

Reply:
will begin using authz module naming in a parallel dev branch :+1: